debian/control

```diff
@@ -13,25 +12,49 @@ Build-Depends:
   cmake
 Standards-Version: 4.5.1
 
-Package: pimodbus-master
+Package: revpi-modbus-client
 Architecture: any
+Section: contrib/electronics
 Depends:
   ${shlibs:Depends},
   ${misc:Depends}
-Description: Revolution Pi Modbus master
+Replaces: pimodbus-master
+Breaks: pimodbus-master
+Description: Revolution Pi Modbus client
  Connect to ModbusRTU or ModbusTCP servers (formerly slaves) from
  this RevPi device.
  Revolution Pi is a set of IoT products based on the
  Raspberry Pi and geared towards industrial usage.
 
-Package: pimodbus-slave
+Package: revpi-modbus-server
 Architecture: any
+Section: contrib/electronics
 Depends:
   ${shlibs:Depends},
   ${misc:Depends}
-Description: Revolution Pi Modbus slave
+Replaces: pimodbus-slave
+Breaks: pimodbus-slave
+Description: Revolution Pi Modbus server
  Use the RevPi device as a ModbusRTU or ModbusTCP server (formerly
  slave). This will allow ModbusRTU or ModbusTCP clients (formerly
  masters) to connect to this RevPi device.
  Revolution Pi is a set of IoT products based on the Raspberry Pi and
  geared towards industrial usage.
+
+Package: pimodbus-master
+Architecture: armhf
```

Comment:
Why armhf?

Shouldn't this be all?
https://wiki.debian.org/RenamingPackages
> Note that the package does not contain any architecture specific files anymore and therefore the Architecture is set to all, even if it was any before.

Reply:
Because of the acceptance criteria of [REVPI-3129]. Since there were no arm64 packages provided before, there shouldn't be transitional packages for it. @naruxde @nbuchwitz Do we change this requirement?

Reply:
And even in Debian there are transitional packages where the old package is not available. Also the user gets the correct package if he follows an older documentation.

Reply:
Changed this back and marked this deviation in the ticket.